plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -212,7 +224,7 @@ json_entry_array(const char *name, int name_len, const char *value, int value_le
 
 /// Helper functions to collect txn information from TSMBuffer
 std::string
-collect_headers(TSMBuffer &buffer, TSMLoc &hdr_loc, int64_t body_bytes)
+collect_headers(TSMBuffer &buffer, TSMLoc &hdr_loc, std::string_view body, int64_t body_length)
```

Comment:
Why pass a `string_view` and a length?

Reply:
If the body is not being collected, then the string_view will be empty. In these cases, the explicit body_length is used to fill in the content::size field without the body content.

Instead I can create an overload, one taking a string_view another taking a length, which each call a common helper function with the 'content' field data. I'll do that.